plugins.go

```diff
@@ -444,7 +442,7 @@ func (j *JSVM) LoadTykJSApi() {
 			r.Header.Set(k, v)
 		}
 		r.Close = true
-		resp, err := client.Do(r)
```

Comment:
I think in future we will use custom client, in order to override TLS settings, but for now make sense to merge for consistency.

Reply:
Yes - also because `http.DefaultClient` may be modified by something else, e.g. a badly written library.